app/assets/stylesheets/common/components/bookmark-list.scss

```
@@ -6,13 +6,21 @@ $mobile-breakpoint: 700px;
   }
 }
 
+.topic-list-data.fill-space {
+  flex: 0 1 45px;
+}
+
 .bookmark-list-item {
   .topic-list-data.post-metadata {
     text-align: center;
   }
   @media (max-width: $mobile-breakpoint) {
-    .main-link {
-      padding-right: 0.5em;
+    .topic-list-data.main-link {
+      padding-right: 1em;
+      flex-direction: column;
+      & ~ .topic-list-data {
```

I think `&` here is superfluous?

```suggestion
      ~ .topic-list-data {
```